### ISSUE TITLE
Stuck on Applying Changes Using Enums

### ISSUE THREAD

**Issue by parakeetberlin** [prisma/prisma#3198]
**Describe the bug**
Stuck on "Applying Changes"

**To Reproduce**
1. Include in your datamodel.prisma and schema.graphl a type that takes consist of an ID and String like so:

datamodel.prisma
```
type Permission {
    id: ID! @unique
    title: String!
}
```

schema.graphql
```
type Permission {
    id: ID!
    title: String!
}
```
2. Create some entries via Mutations in the GraphQL Playground
3. Change the type to use an enum instead of a string for the title like so: 

datamodel.prisma
```
type Permission {
    id: ID! @unique
    title: PermissionEnum!
}
```

schema.graphql
```
type Permission {
    id: ID!
    title: PermissionEnum!
}
```
4. Run ``` prisma deploy --force ```

**Expected behavior**
Changes should happen.

**Screenshots**

**Versions (please complete the following information):**
 - OS: OS X High Sierra 10.13.4
 - `prisma` CLI: `prisma/1.17.1 (darwin-x64) node-v10.10.0`
 - Prisma Server: `1.16.2`
 - `graphql-yoga`

**Additional context**
When adding a second attribute that also uses an enum then it doesn't stock on "Applying Changes"...
like so: 

datamodel.prisma
```
type Permission {
    id: ID! @unique
    title: PermissionEnum!
    test: PermissionEnum
}
```

schema.graphql 
```
type Permission {
    id: ID!
    title: PermissionEnum!
    test: PermissionEnum
}
```

**Comment by marktani** [comment]
added to #2018

**Comment by mavilein** [comment]
Thanks for reporting the issue and your thorough description! 🙏 This has been fixed in `1.18.1`.
